core/src/main/java/io/grpc/internal/BinaryLogProvider.java

```diff
@@ -100,7 +100,7 @@ Channel wrapChannel(Channel channel) {
   /**
    * Wraps a {@link ServerMethodDefinition} such that it performs binary logging if needed.
    */
-  final <ReqT, RespT> ServerMethodDefinition<?, ?> wrapMethodDefinition(
+  public <ReqT, RespT> ServerMethodDefinition<?, ?> wrapMethodDefinition(
```

Comment:
Why remove the `final`? I'd be happy adding it to the client version. But I guess it doesn't matter at all since this will change soon anyway.

Reply:
The `final` is probably a good guard to prevent this from becoming a super interceptor API. I'll keep it until there's a good reason to remove it.